docs/vnode.md

```diff
@@ -0,0 +1,131 @@
+# VNode
+
+A VNode is a representation of a dom node. Most end users will probably have their VNodes generated through [virtual-hyperscript](https://github.com/Matt-Esch/virtual-dom/tree/master/virtual-hyperscript), but understanding the VNode interface can be useful.
+
+In virtual-dom, VNodes are turned from virtual nodes into real nodes with the createElement function. You can read the code at [vdom/create-element](https://github.com/Matt-Esch/virtual-dom/blob/master/vdom/create-element.js)
+
+## Arguments
+
+**tagName**  
+A string, e.g. 'div', 'span', 'article'
+
+**properties** *(optional)*  
+An object that maps property names to their values, e.g. `{ id: "foo", className: "bar" }`
```

Comment:
maybe leave a comment noting that it's properties, i.e. DOM properties, not HTML attributes.